octobot/backtesting/octobot_backtesting.py

```diff
@@ -217,6 +217,17 @@ async def _create_service_feeds(self):
                                   f"might not work properly")
 
     async def _init_exchanges(self):
+        if self.backtesting_config['exchange_type'] != "current_profile":
+            for exchange in self.exchange_type_by_exchange:
+                if self.backtesting_config['exchange_type'] == "SPOT":
```

Comment:
We should use octobot_trading enums `ExchangeTypes` values

Reply:
the current code uses the enums, but still the ifs. How would you remove the ifs?